### ISSUE TITLE
Is there a way to obtain individual request times?

### ISSUE THREAD

**Issue by noftasmos1998** [rycus86/prometheus_flask_exporter#117]
From the metrics provided I'm not understadning how can I obtain the max/min request time in a timeframe given. Is there a way to do this?

**Comment by rycus86** [comment]
I think that can be done using `max_over_time(flask_http_request_duration_seconds[15m])` - see https://prometheus.io/docs/prometheus/latest/querying/functions/#aggregation_over_time

**Comment by noftasmos1998** [comment]
> I think that can be done using `max_over_time(flask_http_request_duration_seconds[15m])` - see https://prometheus.io/docs/prometheus/latest/querying/functions/#aggregation_over_time

Yes exactly how I thought but somehow I don't have that metric available, I only have flask_http_request_duration_seconds_sum,_count and _bucket.

**Comment by rycus86** [comment]
Oh, yes of course, it's a Histogram - sorry. I don't think you can get the max value with the built-in metrics only.
Not even sure how can we get that with Prometheus metrics. 🤔
